src/odemis/gui/cont/multi_point_correlation.py

```diff
@@ -481,11 +502,21 @@ def _do_correlation(self):
 
         # Convert from pixel to physical coordinates
         projected_poi_px = self.correlation_target.correlation_result["output"]["poi"][0]["image_px"]
+
         projected_poi = self.correlation_target.fib_stream.getPhysicalCoordinates(projected_poi_px)
         projected_poi_target = Target(x=projected_poi[0], y=projected_poi[1], z=0, name="PPOI",
                                       type=TargetType.ProjectedPOI,
                                       index=1,
                                       fm_focus_position=0)
+        if self.correlation_target.fib_surface_fiducial:
+
```

Comment:
No need for an empty line just after the iff. But you can put one after the block.